### DISCUSSION TITLE
How to get a new badge on github in 2022?

### DISCUSSION THREAD

**Discussion by emersonxinay** [community/community#34125] [Other Feature Feedback, Questions, & Ideas]
How to get a new badge on github in 2022?
Can you ask in Spanish or only in English?

**Comment by radar-blip** [comment]
Read here for profile [achievement badges](https://github.com/Schweinepriester/github-profile-achievements#achievements).
You can ask in either english or spanish. 👍🏻

**Reply by DarioChiappello** [reply]
Thanks

**Comment by Basitx72** [comment]
Hey! I can communicate with you in English. Are you referring about profile achievements or badges?
If you're looking for.readme badges, visit this page: https://[readme] .so]
(https://readme.so/)/  or Go to https://github.com/Schweinepriester/github-profile-accomplishments to discover more about GitHub profile achievements. I hope this was helpful.

**Comment by Anmol-Baranwal** [comment]
First of all, there are different badges and each one has its unique way =>

You can see the list of badges [here](https://github.com/drknzz/GitHub-Achievements) or [here](https://github.com/Schweinepriester/github-profile-achievements)

List of Badges -
- Starstruck
- Quickdraw
- Pull Shark
- Pair Extraordinaire &nbsp;&nbsp;&nbsp;&nbsp; Visit [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) for more detail
- Galaxy Brain
- YOLO
- Public Sponsor

If you have any doubt regarding this, feel free to ask.

**Note**: Don't try to test the things like Pair Extraordinaire in the private testing repository. It will count and will be inaccessible if you delete the repository. Since it is due to the private repository, it will not have much weight.

**Reply by Anmol-Baranwal** [reply]
@emersonxinay please mark this as an answer if you have no further queries so that others can know that the question has been answered.
